azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java

```diff
@@ -41,21 +51,13 @@ public JavaVersion getJavaVersion() {
     public RuntimeStack getLinuxRuntime() throws MojoExecutionException {
         // todo: add unit tests
         if (StringUtils.equalsIgnoreCase(javaVersion, JRE_8)) {
-            if (StringUtils.isEmpty(webContainer)) {
-                return RuntimeStack.JAVA_8_JRE8;
-            }
-
-            switch (webContainer.toLowerCase(Locale.ENGLISH)) {
-                case TOMCAT_8_5:
-                    return RuntimeStack.TOMCAT_8_5_JRE8;
-                case TOMCAT_9_0:
-                    return RuntimeStack.TOMCAT_9_0_JRE8;
-                case WILDFLY_14:
-                    return RuntimeStack.WILDFLY_14_JRE8;
-                default:
-                    throw new MojoExecutionException(
-                        String.format("Unknown value of <webContainer>. Supported values are %s, %s and %s.",
-                            TOMCAT_8_5, TOMCAT_9_0, WILDFLY_14));
+            final String fixWebContainer = StringUtils.isEmpty(webContainer) ? JRE_8 : webContainer;
```

Comment:
`fixWebContainer` needs to get better naming, cuz JRE_8` is not a web container

Reply:
changed it to `fixRuntime`